[debug]输出结果总是会包含“[ERROR] name 'data' is not defined”

## **Describe the bug [描述]**
输出结果总是会包含“[ERROR] name 'data' is not defined”


## **To Reproduce [复现]**

### version info [基本信息]

* DDNS Version: 2.97
* OS Version： win10 2004
* Type(运行方式): Binary


### config file [配置文件]
```json
{
  "$schema": "https://ddns.newfuture.cc/schema/v2.8.json",
  "debug": true,
  "dns": "callback",
  "id": "https://ipv6.dynv6.com/api/update?ipv6=auto&zone=__DOMAIN__&token=**********&ipv6=__IP__",
  "index4": "false",
  "index6": "public",
  "ipv4": [],
  "ipv6": [
    "***************"
  ],
  "proxy": null,
  "token": "",
  "ttl": null
}
```

###  debug output [调试输出]
```sh
DDNS[ v2.9.7@2020-09-11T03:25:49+00:00 ] run: nt win32
Configuration was loaded from <== C:\DDNS\config.json
========================= Sun Feb  7 02:20:38 2021 =========================
2021-02-07 02:20:38,026 <cache.load> 48@util\cache.py
[DEBUG] load cache data from C:\Users\STEVES~1\AppData\Local\Temp\ddns.cache
2021-02-07 02:20:38,027 <run.main> 189@run.py
[WARNING] Cache file is out of dated.

*********************(AAAA) ==> ********************************** [via DIRECT]
2021-02-07 02:20:38,074 <callback.update_record> 90@dns\callback.py
[INFO] >>>>>********************(AAAA)
2021-02-07 02:20:38,075 <callback.request> 40@dns\callback.py
[INFO] ipv6.dynv6.com//api/update : {'ipv6': '**************************************', 'zone': '*********************', 'token': '*******************'}
2021-02-07 02:20:39,348 <callback.request> 73@dns\callback.py
[DEBUG] /api/update?ipv6=********************************************&zone=****************************&token=**************************************** : result:addresses updated
2021-02-07 02:20:39,348 <run.change_dns_record> 125@run.py
[ERROR] name 'data' is not defined
2021-02-07 02:20:39,351 <cache.sync> 85@util\cache.py
[DEBUG] save cache data to C:\Users\STEVES~1\AppData\Local\Temp\ddns.cache
```

--- a/dns/callback.py
+++ b/dns/callback.py
@@ -71,7 +71,7 @@ def request(method, action, param=None, **params):
         raise Exception(res)
     else:
         debug('%s : result:%s', action, res)
-        return data
+        return res
 
 def replace_params(domain, record_type, ip, params):
     """

